docs/11-Codecoverage.md

```diff
@@ -103,6 +103,24 @@ codecept run --coverage --coverage-xml --coverage-html
 XML and HTML reports are stored to the `_output` directory. The best way to review report is to open `index.html` from `tests/_output/coverage` in your browser.
 XML clover reports are used by IDEs (like PHPStorm) or Continuous Integration servers (like Jenkins).
 
+### Advanced configuration
+We support setting any configuration on the `CodeCoverage` object. Note that while we expose all configuration
+options here, not all combinations of them have been tested with CodeCeption. Using a non-standard configuration may break things.
+
+{% highlight yaml %}
+
+coverage:
+  advanced:
+    forceCoversAnnotation: true
+    checkForMissingCoversAnnotation: true
+    
+{% endhighlight %}
+
+The code doing the configuration is fully dynamic meaning that the actual options available depend on the version of `php-code-coverage` you are using.
+Some of the available options are documented in PHPUnit's docs: https://phpunit.readthedocs.io/en/9.1/code-coverage-analysis.html.
```

Comment:
I don't see any examples of available options in that page.

Reply:
Ill double check

Reply:
9.2 is the latest version now - https://phpunit.readthedocs.io/en/9.2/code-coverage-analysis.html
It is still unclear what are the possible options.